rs/bitcoin/ckbtc/minter/src/lib.rs

```diff
@@ -1323,6 +1362,21 @@ impl CanisterRuntime for IcCanisterRuntime {
         .map(|(result,)| result.signature)
         .map_err(|err| CallError::from_cdk_error("sign_with_ecdsa", err))
     }
+
+    async fn send_transaction(
+        &self,
+        transaction: &tx::SignedTransaction,
+        network: Network,
+    ) -> Result<(), CallError> {
+        ic_cdk::api::management_canister::bitcoin::bitcoin_send_transaction(
```

Comment:
Is this using the deprecated Bitcoin integration API?!? (The Bitcoin canister should **not** be called through the management canister)

Reply:
Yes, but I don't want to modify the old code. I also reverted the change to management.rs, so this call now just redirects to there.

We can cleanup when we upgrade ic_cdk.